fitbenchmarking/cli/main.py

```diff
@@ -0,0 +1,109 @@
+"""
```

Comment:
It wasn't intended to be, although I could change it if it would be better

Reply:
Well I'm not entirely sure what the convention is when executing with a CLI, but if you were executing with `python -m` if would run `__main__.py`, so I was wondering if it was an idea to be consistent with this.

Thinking about it a little more, what is the reason why `main.py` is in the cli subpackage?  Is it actually necessary?  As otherwise you could just put it in the directory above and name it `__main__`.  Then you could also run fitbenchmarking with `python -m`.

Reply:
Yes, I guess it depends on what we want to do with this, if we want any other CLI commands, moving it out would be counter-productive. The reason I made a CLI dir is as an analogue to the bin dir, as well as separating it from the core code that we already have in fitbenchmarking.

Would it be better to add an `__main__.py` which is a pass-through to this?

Reply:
Agreed, that would probably make the most sense.  I think it would be worth doing that, as I think generally having a cli implies being executable as a module too.  It would also allow debugging with `python -m pdb -m fitbenchmarking` (at least when we support python 3.7).